docs/logging.md

```diff
@@ -17,8 +17,8 @@ As of Kubernetes 0.11, when you create a cluster the console output reports the
 a URL for a [Kibana](http://www.elasticsearch.org/overview/kibana/) dashboard viewer for the logs that have been ingested
 into Elasticsearch.
 ```
-Cluster logs are ingested into Elasticsearch running at https://130.211.152.93/api/v1beta1/proxy/services/elasticsearch-logging/
-Kibana logging dashboard will be available at https://130.211.152.93/api/v1beta1/proxy/services/kibana-logging/ (note the trailing slash)
+Elasticsearch is running at https://104.197.10.10/api/v1beta3/proxy/namespaces/default/services/elasticsearch-logging
```

Comment:
This is a direct copy of the output of `kubectl cluster-info`. 

I think we need the trailing slash if we want to `curl` the proxy.